IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -209,7 +209,7 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
             
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->

Reply:
No change needed for changelog